CONTRIBUTING.md

```diff
@@ -207,6 +207,7 @@ You must have the following installed on your system to contribute locally:
 - [`Node.js`](https://nodejs.org/en/) (See the root [.node-version](.node-version) file for the required version. You can find a list of tools on [node-version-usage](https://github.com/shadowspawn/node-version-usage) to switch the version of [`Node.js`](https://nodejs.org/en/) based on [.node-version](.node-version).)
 - [`yarn`](https://yarnpkg.com/en/docs/install)
 - [`python`](https://www.python.org/downloads/) (since we use `node-gyp`. See their [repo](https://github.com/nodejs/node-gyp) for Python version requirements.)
+  - Note for Debian-based systems: `python` is pre-installed.<br>`sudo apt install g++ make cmake` meets the additional requirements to run `node-gyp` in the context of building Cypress from source.
```

Comment:
```suggestion
  - Note for Debian-based systems: `python` is pre-installed.<br>`sudo apt install g++ cmake` meets the additional requirements to run `node-gyp` in the context of building Cypress from source.
```

Sorry for this. One more correction, pointed out by @alexsch01: If `cmake` is installed, it also installs `make`.

Reply:
Actually nope, make should still be in the install command

Sorry for the confusion

Reply:
I've hidden the latest suggestion. I'm not an `apt` expert. I have just been checking that the instructions work on Ubuntu.